src/lib/error/from-legacy-error.ts

```diff
@@ -21,14 +21,14 @@ const getStatusCode = (errorName: string): number => {
             return 400;
         case 'MinimumOneEnvironmentError':
             return 400;
+        case 'InvalidOperationError':
+            return 400;
```

Comment:
InvalidOperation should not be related to authorization, so not 403, but rather 400

Reply:
We use it for features behind a feature flag. We had a discussion about it and agreed on 403. Since it's forbidden behind a feature flag. I wouldn't change this one.

Reply:
We either need to change it to something else than 403 or move alot of things away from InvalidOperationError.
We are using 403 for these and it does not make any sense.

```
You can not have an empty string for stickiness.
You can not delete the default project!
You cannot change built in roles.
You can not have an empty string for stickiness.
```

and this is just start of the list. these are not 403s.

If we want to have features that are behind a feature flag, maybe the error should be more explicit by name or something.